zboxcore/sdk/allocation.go

```diff
@@ -1377,6 +1381,22 @@ func (a *Allocation) GetMinWriteRead() (minW float64, minR float64, err error) {
 	return minWritePrice, minReadPrice, nil
 }
 
+func (a *Allocation) GetMaxStorageCostFromBlobbers(size int64, blobbers []*BlobberAllocation) (float64, error) {
+	var cost common.Balance // total price for size / duration
+
+	for _, d := range blobbers {
+		fmt.Printf("write price for blobber %f datashards %d parity %d\n",
```

Comment:
please don't use fmt.Printf to print log. It is impossible to turn off in config file.